src/pykeen/models/predict.py

```diff
@@ -141,66 +244,56 @@ def get_tail_prediction_df(
     ...     model='RotatE',
     ... )
     >>> df = get_tail_prediction_df(result.model, 'brazil', 'accusation', triples_factory=result.training)
-    """
-    head_id = triples_factory.entity_to_id[head_label]
-    relation_id = triples_factory.relation_to_id[relation_label]
-    batch = torch.as_tensor([[head_id, relation_id]], dtype=torch.long, device=model.device)
-    scores = model.predict_t(batch, mode=mode)
-    scores = scores[0, :].tolist()
-    rv = pd.DataFrame(
-        [
-            (entity_id, entity_label, scores[entity_id])
-            for entity_label, entity_id in triples_factory.entity_to_id.items()
-        ],
-        columns=["tail_id", "tail_label", "score"],
-    ).sort_values("score", ascending=False)
 
-    return _postprocess_prediction_df(
-        rv,
-        add_novelties=add_novelties,
-        remove_known=remove_known,
-        testing=testing,
-        training=triples_factory.mapped_triples,
-        query_ids_key="tail_id",
-        col=2,
-        other_col_ids=(head_id, relation_id),
+    The optional `tails` parameter can be used to restrict prediction to a subset of entities, e.g.
+    >>> df = get_tail_prediction_df(
+    ...     result.model,
```

Comment:
need to double check these examples, there was way too much mixing of variadic arguments and lots of stuff got missed/ orders were invalid